cfme/tests/cloud/test_snapshot.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme import test_requirements
+from cfme.cloud.provider.openstack import OpenStackProvider
+
+
+@pytest.mark.manual
+@pytest.mark.provider([OpenStackProvider])
+@test_requirements.snapshot
+def test_osp_snapshot_buttons():
+    """
+    Test new OSP snapshot button and make sure Snapshot link is removed from Instance Details
+    page.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Cloud
+        caseimportance: medium
+        initialEstimate: 1/4h
+        startsin: 5.10
+        setup:
+            1. Have OSP provider added and test instance created
+        testSteps:
+            1. Navigate to test instance
+            2. Make sure the snapshot button is displayed in the top menu
+            3. Try snapshot crud
+            4. Make sure the old Snapshot link is NOT displayed in the instance summary page
```

Comment:
This is more an expectedResult than a testStep, what do you think?

Reply:
Two things:
1. I have more than 200 manual testcases to move. If we are going to debate every test step, this is going to become very problematic. I don't want to sound negative, but if the test step or expected result makes sense, then I would leave it as is and move on.
2. In this example, it can be considered a test step. You have to check that the link is not displayed in that page. Expected result then would be that you don't see that link. With my first point in mind, I would leave this as is.

Reply:
@apagac, understood on frustration in moving all of these cases.

In this case, the step and result are about the same, can you please update the step to state where the user should be in navigation? What page should they be on, looking for the snapshot link (that is not displayed).  Think of a new user taking over this test case - they might not know where to be looking for that link.

Reply:
Tried to make step 4 more clear. Please let me know if additional change is required.